recipes/clingo-dl/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "clingo-dl" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/potassco/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: daceed282ae46f443b0423be4f6e68b9559159639522dbdcb1aae6a623566b4d
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
```

Comment:
```suggestion
```

Reply:
I removed this. Actually, this was just some legacy cruft that was necessary in the past to avoid building with an unsupported C compiler on windows pulled in by python 2.7.